pkg/sql/plan/function/func_binary.go

```diff
@@ -702,75 +702,77 @@ func ConvertTz(ivecs []*vector.Vector, result vector.FunctionResultWrapper, proc
 		fromTz, null2 := fromTzs.GetStrValue(i)
 		toTz, null3 := toTzs.GetStrValue(i)
 
-		if fromTz == nil || toTz == nil {
-			return moerr.NewInvalidArg(proc.Ctx, "ConvertTz", "date zone is empty")
-		}
-
 		if null1 || null2 || null3 {
-			if err = rs.AppendBytes(nil, true); err != nil {
+			err = rs.AppendBytes(nil, true)
+			return err
```

Comment:
it cannot return the error directly if the error was not nil.

this will cause the result has a different length with the input columns.

for example. 
one of input column is [value1, null, value2].
and result will be [result1, null], and error is nil. (lost the third result)

Reply:
I fix this, plz check.